include/psa/crypto_values.h

```diff
@@ -1440,6 +1440,19 @@
  */
 #define PSA_KEY_LIFETIME_PERSISTENT             ((psa_key_lifetime_t)0x00000001)
 
+/** The minimum value for a key identifier chosen by the application.
+ */
+#define PSA_KEY_ID_USER_MIN                     ((psa_key_id_t)0x00000001)
+/** The minimum value for a key identifier chosen by the application.
```

Comment:
Maximum*

Reply:
Maximum would be good. Minimal/Maximal; Minimum/Maximum.

Reply:
Thanks. Fixed, also application → implementation.